Templates/DefaultProject/Template/ExportScripts/export_standalone_monolithic_engine_centric.py

```diff
@@ -0,0 +1,161 @@
+#
+# Copyright (c) Contributors to the Open 3D Engine Project.
+# For complete copyright and license terms please see the LICENSE at the root of this distribution.
+#
+# SPDX-License-Identifier: Apache-2.0 OR MIT
+#
+#
+
+
+import argparse
+import glob
+import logging
+import os
+import o3de.export_project as exp
+import pathlib
+import sys
+import time
+
+from o3de.export_project import process_command
+from o3de import manifest
+
+project_json_data = manifest.get_project_json_data(project_path=o3de_context.project_path)
+project_name = project_json_data.get('project_name')
+host_asset_platform = exp.determine_host_asset_platform()
+parser = argparse.ArgumentParser(prog='Exporter for standalone builds',
+                                 description = "Exports a project as standalone to the desired output directory with release layout. "
+                                                "In order to use this script, the engine and project must be setup and registered beforehand. ")
+parser.add_argument('-out', '--output-path', type=pathlib.Path, required=True, help='Path that describes the final resulting Release Directory path location.')
+parser.add_argument('-cfg', '--config', type=str, default='profile', choices=['release', 'profile'],
+                        help='The CMake build configuration to use when building project binaries. If tool binaries are built with this script, they will use profile mode.')
+parser.add_argument('-aof', '--archive-output-format',
+                        type=str,
+                        help="Format of archive to create from the output directory",
+                        choices=["zip", "gzip", "bz2", "xz"], default="zip")
+parser.add_argument('-sl', '--seedlist', type=pathlib.Path, dest='seedlist_paths', action='append',
+                        help='Path to a seed list file for asset bundling. Specify multiple times for each seed list.')
+parser.add_argument('-gf', '--gem-folder-name-to-copy', type=str, dest='gem_folder_names_to_copy', action='append',
+                        help="The name of a Gem folder to include in release. Only specify folder name. This should match what is in the project directory.")
+parser.add_argument('-pfp', '--project-file_pattern-to-copy', type=str, dest='project_file_patterns_to_copy', action='append',
+                        help="Any additional file patterns located in the project directory. File patterns will be relative to the project path.")
```

Comment:
IMHO, specifying a package manifest file is a better experience than to have to specify it on a command line every time i need to build an image. I think a manifest can also contain other arguments that are contained in this arg list. Thats not to say that we remove the arguments, but the manifest can contain the default values and the arguments can be a way to override them if needed.

Reply:
does o3de already have a mechanism for a package manifest file, or would we have to create one specifically for this process? If the latter, that may be better done in a dedicated PR.

Reply:
Maybe manifest is not the right word, but more like a packaging rule. (Think of CMake presets, but per project). It would be directly related to this process, but if you want to do another PR for it, then this script will need to be updated accordingly. But probably better to think more generally about this problem and create an RFC + Design on the packaging process.

Reply:
Agreed, an RFC would be warranted for that. I think it's out of scope for the PR at present, but I'll file an issue related to this.